tests/integration/pybaseball/test_standings.py

```diff
@@ -45,3 +45,10 @@ def test_standings_future(self) -> None:
         standings_list = standings(season)
 
         assert standings_list == []
+
+    def test_standings_none(self) -> None:
+        season = None
+
+        standings_list = standings(season)
+
+        assert standings_list == []
```

Comment:
Can you explain the rationale here?

Reply:
This was already part of the range test above, but was acting funky. Running it again today it's back to working. In theory the None should get replaced with a sane default, but between our func and the actual call, it's not behaving, so I'm removing it.